README.md

```diff
@@ -34,32 +34,9 @@ Build an API Endpoint for each of the following, and connect it to your React Ap
 
 Upon your first commit, please submit a Pull Request and add _both_ the **Trello Set Up** and **Backend MVP Features** Task lists to your first Pull Request comment:
 
-```markdown
-## Trello Set Up:
-* [ ] Use your existing Trello account from the Front End Project, or create a new one
-* [ ] Create a new board called "Lambda Notes(Backend) - {Your Name}"
-* [ ] Create lists titled `backlog`,`To Do`, `In Progress`, and `Done`
-* [ ] Fill in the `To Do` list with the MVP features listed below
-* [ ] Fill in the `backlog` list with all the extra features listed below
-* [ ] Share your board with the project manager that has been assigned to you.  If you have not been assigned yet, reach out to your Section Lead for guidance
-* [ ] Add your Trello URL to your project's README.md file.  Commit the change, push it to your repository & submit a pull request
-
-## Backend MVP Features:
-* [ ] Create a Database for your app
-* [ ] Create a Node app and connect it to your database
-* [ ] Create a Registration Page that allows users to create accounts for your app and sign in with email/password
-* [ ] Store these User Accounts in your Database
-* [ ] Add a Sign In Page
-* [ ] Store Notes in your Database
-* [ ] Deploy your application to the web
-
-Build an API Endpoint in your Node App for each of the following, and connect them to your React App. These should all be read from your Database and scoped to the logged in user. Write tests for your endpoints.
-* [ ] Display a list of notes
-* [ ] Create a note with a title and content
-* [ ] View an existing note
-* [ ] Edit an existing note
-* [ ] Delete an existing note
-```
+***
+# Trello Board Link
+`https://trello.com/b/1BI5xT2N`
```

Comment:
With markdown syntax, in particular, there is a GitHub Flavored markdown, to make this a hypertext link, you can put a description in square brackets and the link following it in parenthesis, e.g.

```md
[description of awesome website](http://awesomewebsite.com)
```

[Peter's amazing Trello board for this project](https://trello.com/b/1BI5xT2N)

You can also use `#` octothorpe's as headers, e.g. for an `<h1>` use one octothorpe, or for an `<h5>` header, use five

```md
# [Click here for Peter's amazing Trello board for this project](https://trello.com/b/1BI5xT2N)

##### [Click here for Peter's amazing Trello board for this project](https://trello.com/b/1BI5xT2N)

```

# [Click here for Peter's amazing Trello board for this project](https://trello.com/b/1BI5xT2N)

##### [Click here for Peter's amazing Trello board for this project](https://trello.com/b/1BI5xT2N)

Reply:
https://help.github.com/articles/basic-writing-and-formatting-syntax/